tests/migration.go

```diff
@@ -4,11 +4,14 @@ import (
 	"fmt"
 	"time"
 
+	expect "github.com/google/goexpect"
```

Comment:
This does not belong to this group (test framework).
I suggest you organize the imports per domain.

Reply:
Done